src/main/java/me/leoko/advancedban/utils/SQLQuery.java

```diff
@@ -7,17 +7,20 @@
  */
 public enum SQLQuery {
     CREATE_TABLE_PUNISHMENT(
-            "CREATE TABLE IF NOT EXISTS `Punishments` ("+
-            "`id` int NOT NULL AUTO_INCREMENT," +
-            "`name` VARCHAR(16) NULL DEFAULT NULL," +
-            "`uuid` VARCHAR(35) NULL DEFAULT NULL," +
-            "`reason` VARCHAR(100) NULL DEFAULT NULL," +
-            "`operator` VARCHAR(16) NULL DEFAULT NULL," +
-            "`punishmentType` VARCHAR(16) NULL DEFAULT NULL," +
-            "`start` LONG DEFAULT NULL," +
-            "`end` LONG DEFAULT NULL," +
-            "`calculation` VARCHAR(50) NULL DEFAULT NULL," +
-            "PRIMARY KEY (`id`))",
+            "CREATE TABLE IF NOT EXISTS `Punishments` (" +
+            "`id` INT NOT NULL AUTO_INCREMENT," +
+            "`name` VARCHAR(16) NOT NULL," +
+            "`uuid` VARCHAR(35) NOT NULL," +
```

Comment:
You should set this to VARCHAR(36) since that's the standard.

Reply:
Or 32. As the plugin stores the UUIDs without dashes anyways.

I’ll check the code to see what’s better.